oauth2client/contrib/django_util/__init__.py

```diff
@@ -236,16 +304,41 @@ def __init__(self, settings_instance):
             _get_oauth2_client_id_and_secret(settings_instance)
 
         if ('django.contrib.sessions.middleware.SessionMiddleware'
-                not in settings_instance.MIDDLEWARE_CLASSES):
-            raise exceptions.ImproperlyConfigured(
-                "The Google OAuth2 Helper requires session middleware to "
-                "be installed. Edit your MIDDLEWARE_CLASSES setting"
-                " to include 'django.contrib.sessions.middleware."
-                "SessionMiddleware'.")
+           not in settings_instance.MIDDLEWARE_CLASSES):
+                raise exceptions.ImproperlyConfigured(
+                  "The Google OAuth2 Helper requires session middleware to "
+                  "be installed. Edit your MIDDLEWARE_CLASSES setting"
+                  " to include 'django.contrib.sessions.middleware."
+                  "SessionMiddleware'.")
+        (self.storage_model, self.storage_model_user_property,
+         self.storage_model_credentials_property) = _get_storage_model()
 
 
 oauth2_settings = OAuth2Settings(django.conf.settings)
 
+_CREDENTIALS_KEY = 'google_oauth2_credentials'
+
+
+def get_storage(request):
+    """ Gets a Credentials storage object for the Django OAuth2 Helper object
```

Comment:
Perhaps: 
```
""""Gets the credential storage provided by the Django OAuth2 helper.

This can be backed by the session or backed by a user model depending on
the configuration.

Args:
    request:  The current Django request.

Returns:
    An `oauth2client.client.Storage` object.
```

(Please use napoleon/google style docstrings)